### DISCUSSION TITLE
Is there a Theia browser compatibility page?

### DISCUSSION THREAD

**Discussion by anavarre** [eclipse-theia/theia#10277] [General] [answered]
I would find it helpful to have a browser compatibility section at https://github.com/eclipse-theia/theia so people would immediately know which browser to use and which one is preferred when working with Theia.

Looking up this information I only found https://github.com/theia-ide/sprotty/wiki/Browsers-and-Compatibility which doesn't really help because it's very outdated.

So I guess my questions are as follows:

* Which browsers are supported among Chrome, Safari, Firefox, Vivaldi, Explorer, Edge, Konqueror, etc?
* How can we know the limitations for each browser?
* Can we agree to update the project's main README file to indicate browser compatibility?

Thanks!

**Comment by vince-fugnitto** [comment, marked as answer]
@anavarre thank you for the discussion, while I do think it might be useful, the effort might be too substantial to test and confirm that all the browsers you listed are compatible which each version of Theia.

> Which browsers are supported among Chrome, Safari, Firefox, Vivaldi, Explorer, Edge, Konqueror, etc?

Ideally all should, especially all Chromium-based browsers (as Electron (desktop apps)) use Chromium under the hood.

> How can we know the limitations for each browser?

I would assume only through testing, which would be a community effort.

> Can we agree to update the project's main README file to indicate browser compatibility?

Probably not the readme directly as it will become too noisy, but either under [docs/](https://github.com/eclipse-theia/theia/tree/master/doc) and referenced in the readme, or on the [theia-website](https://github.com/eclipse-theia/theia-website). Please feel free to propose an initial pull-request detailing the compatibility findings you found.

**Reply by anavarre** [reply]
> Ideally all should, especially all Chromium-based browsers (as Electron (desktop apps)) use Chromium under the hood.

I think that's the takeaway for me.

> Probably not the readme directly as it will become too noisy, but either under docs/ and referenced in the readme, or on the theia-website. Please feel free to propose an initial pull-request detailing the compatibility findings you found.

Sure, I'll bake something.

Thanks!
